util/LoggerWOptionsDB.cpp

```diff
@@ -0,0 +1,178 @@
+#include "LoggerWOptionsDB.h"
```

Comment:
not a fan of initialization in general, and this one saves only a few characters...